src/parser/parse_error.ml

```diff
@@ -39,10 +39,22 @@ let no_trailing_whitespace_in_verbatim : Location.span -> Error.t =
   Error.make "'v}' must be preceded by whitespace"
 
 let only_one_title_allowed : Location.span -> Error.t =
-  Error.make "only one title-level heading is allowed"
+  Error.make "only one title-level heading {0 ...} is allowed"
 
-let sections_not_allowed : Location.span -> Error.t =
-  Error.make "sections not allowed in this comment"
+let page_heading_required : string -> Error.t =
+  Error.filename_only "pages must start with a heading"
```

Comment:
Could you add two tests for this: one testing the string `"foo"` (just a paragraph) in context "`All", and one testing `"foo\n{0 Bar}"` also in context "`All"?

I think this will cover

> #### Required heading in pages
>
> ```
> Some paragraph.
>
> Error: pages must start with a heading
> ```

Reply:
I thought I’ve covered this case, thanks for spotting this. I’ll add the test cases ASAP.